src/script/interpreter.cpp

```diff
@@ -1142,24 +1142,24 @@ class CTransactionSignatureSerializer {
 
 uint256 GetPrevoutHash(const CTransaction& txTo) {
     CHashWriter ss(SER_GETHASH, 0);
-    for (unsigned int n = 0; n < txTo.vin.size(); n++) {
-        ss << txTo.vin[n].prevout;
+    for (const auto& txIn : txTo.vin) {
```

Comment:
@luke-jr I think I get it now. `CTransactionRef &txOut` and `CTransactionRef& txIn` if used in other parts of the code base. The suggestions is that I change all `txIn`/`txOut` to `txin`/`txout` in my PR?